spec/lcm/slow/nfs_release_table_spec.rb

```diff
@@ -0,0 +1,167 @@
+require 'pp'
```

Comment:
Duplicates `lcm_end_to_end_spec.rb` which will substantially prolong the build. Am I guessing right that your plan is to drop `lcm_end_to_end_spec.rb` soon? 🔮

Reply:
That's why it's in the slow test folder. I think we'll need e2e tests to cover both ADS and NFS lcm logic - currently, everyone uses ADS in production, so our main test should be based on that. In the future, IMHO, the logic in `lcm_end_to_end_spec.rb` should be based on NFS and there should be a `slow/ads_release_table_spec.rb` instead.